cmd/ocm/create/cluster/cmd.go

```diff
@@ -75,6 +80,12 @@ func init() {
 
 	arguments.AddProviderFlag(fs, &args.provider)
 	arguments.AddCCSFlags(fs, &args.ccs)
+
+	fs.Var(
+		&args.gcpServiceAccount,
```

Comment:
Ideally declaring this flag would be part of `arguments.AddCCSFlags()`.  
One way to do that that might be move the FilePath var inside CCS struct, and making `constructGCPCredentials` a method on CCS struct.  Don't know if worth it.

Reply:
Opened #199 for some validations we should improve.